flocx_market/db/api.py

```diff
@@ -0,0 +1,63 @@
+import pymysql
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from flocx_market.db import models as db_models
+
+
+_ENGINE = None
+_SESSION_MAKER = None
+
+#1.connecting
+def get_engine():
+    global _ENGINE
+    if _ENGINE is not None:
+        return _ENGINE
+
+    _ENGINE = create_engine('mysql+pymysql://flocx_market:qwerty123@127.0.0.1:3306/flocx_market', echo=True)
```

Comment:
In the future we will want a mechanism that allows us to specify the database url outside of the source.